packages/components/pie-button/src/button.scss

```diff
@@ -7,35 +7,32 @@
     font-weight: dt.$font-interactive-m-weight;
 }
 
-// TODO - Use latest design tokens for font size and line height
-//        once fozzie has been updated to handle them.
-
 .o-btn--xsmall {
     padding: 6px dt.$spacing-b;
     min-height: 32px;
-    font-size: 14px;
-    line-height: 20px;
+    font-size: dt.$font-size-14 * 1px;
+    line-height: dt.$font-size-14-line-height * 1px;
 }
 
 .o-btn--small {
     padding: 8px dt.$spacing-d;
     min-height: 40px;
-    font-size: 19px;
-    line-height: 28px;
+    font-size: dt.$font-size-19 * 1px;
```

Comment:
is this the best way to use design tokens for font-sizes?

Reply:
I think so, open to other options though - we made the fonts unitless in `pie-design-tokens` so it's easier to run calc functions on them. means adding the `px` in the variable like above 😄

Reply:
I think in the longer term, we'll have a utility function for this anyway, as we'll be converting px to rem & px